Workbooks/Intune Audit/Audit/IntuneAuditActivity.workbook

```diff
@@ -0,0 +1,390 @@
+{
+  "version": "Notebook/1.0",
+  "isLocked": false,
```

Comment:
for sure you want this to be `"isLocked": true,` (or just remove the line entirely)